src/ui/Interface.js

```diff
@@ -2,7 +2,8 @@ module.exports = function Interface (options) {
 
     options.latId = options.latId || 'lat';
     options.lngId = options.lngId || 'lng';
-    options.selector = options.selector || 'geo_location'
+    options.selector = options.selector || 'geo_location';
+    options.locationText = options.locationText || 'location';
```

Comment:
This is great. Could we say `placenameId`?

Also, I think there's ambiguity between `geo_location`/`options.selector` and `location`/`locationText` -- can we standardize these to be more understandable? What about:

```js
options.placenameInputId = options.placenameInputId || 'placenameInput'; // the placename as input by the user
options.placenameDisplayId = options.placenameDisplayId || 'placenameDisplay'; // the placename as will be stored/displayed
```